modules/find-failing-cis-controls.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-cluster-configuration.adoc
+:_module-type: PROCEDURE
+[id="find-failing-cis-controls_{context}"]
+= Finding failing CIS controls
+
+[role="_abstract"]
+Similar to the *Policy Violations* sunburst chart in the *Configuration Management* view, the CIS controls widget provides information about failing CIS controls.
+
+Each level of the chart is represented by one ring or circle.
+
+* The innermost circle represents the percentage of failing controls.
+* The next ring represents the control categories.
+* The outermost ring represents individual controls in a particular category.
+
+.Procedure
+. Select *CIS Docker v1.2.0* on the CIS controls widget header.
+Use this to switch between CIS Docker and Kubernetes controls.
+. Hover over the sunburst chart to view details about failing controls.
+. Select *_n_ controls failing* (where _n_ is a number) to view detailed information about failing controls.
+The *Controls* view displays a list of failing controls filtered based on the compliance state.
+. Select a row to view more details, including control descriptions and nodes where the controls are failing.
+The details are visible in a panel.
+. The *Control Findings* section in the information panel lists nodes where the controls are failing.
+Select a row to view more details, including Kubernetes labels, annotations, and other metadata.
+
+Use the detailed information to focus your attention on a subset of nodes, industry standards, or failing controls, and assess, check, and report on the compliance status of your containerized infrastructure.
```

Comment:
You could split this sentence into two, to facilitate localisation. For example:

"You can use the detailed information to focus on a subset of nodes, industry standards, or failing controls. You can also assess, check, and report on the compliance status of your containerized infrastructure."